packages/i18n/package.json

```diff
@@ -34,6 +34,7 @@
     "build:es:watch": "npm run build:es -- -w"
   },
   "dependencies": {
+    "@babel/runtime": "7.1.5",
```

Comment:
if you use an async function without having this as a dependency, then rollup will complain. we don't bundle dependencies with rollup anyways, so this won't increase bundle size